app/controllers/organizations/invitations_controller.rb

```diff
@@ -1,4 +1,8 @@
 class Organizations::InvitationsController < Devise::InvitationsController
+  include OrganizationScopable
+  # TODO: remove next line once we fix links in the invitation emails
+  skip_before_action :set_tenant, except: [:new, :create]
```

Comment:
After tweaking the ActsAsTenant setup in our tests, one of the tests start failing. There was an actual bug and `StaffAccount` kept linking itself to the organization with an id of 1. Including `OrganizationScopable` fixes that up, but we should probably drop the default value from the staff_account.organization_id column so we don't miss similar issues down the road.